assets/js/modules/analytics/datastore/properties.js

```diff
@@ -151,10 +151,12 @@ const baseActions = {
 				return;
 			}
 
-			registry.dispatch( STORE_NAME ).setPropertyID( propertyID );
-
 			if ( PROPERTY_CREATE === propertyID ) {
-				registry.dispatch( STORE_NAME ).setProfileID( PROFILE_CREATE );
+				registry.dispatch( STORE_NAME ).setSettings( {
+					propertyID,
+					profileID: PROFILE_CREATE,
```

Comment:
I think it would make sense to also set `internalWebPropertyID` to `''` here. Otherwise, it might keep its previous value (let's say a property was selected before, but then the user chooses "Create new property"), but since it always should depend on `propertyID`, that would be incorrect.

Reply:
Yes, you are right. I have added internalWebPropertyID here as well.